apps/manager/src/lib/Wagmi.ts

```diff
@@ -1,15 +1,52 @@
-import { Porto } from '@porto/apps'
+import { Env, Porto as PortoConfig } from '@porto/apps'
+import { Dialog, Mode, Porto } from 'porto'
 import { createConfig, createStorage, http } from 'wagmi'
 import { base, baseSepolia, odysseyTestnet } from 'wagmi/chains'
 import { injected } from 'wagmi/connectors'
 
-export const porto = Porto.porto
+const env = Env.get()
+const host = PortoConfig.dialogHosts[env]
+
+export type ModeType =
+  | 'contract'
+  | 'iframe-dialog'
+  | 'inline-dialog'
+  | 'popup-dialog'
+  | 'relay'
+
+export const modes = {
+  contract: Mode.contract(),
+  'iframe-dialog': Mode.dialog({
+    host,
+  }),
+  'inline-dialog': Mode.dialog({
+    host,
+    renderer: Dialog.experimental_inline({
+      element: () => document.getElementById('porto')!,
+    }),
+  }),
+  'popup-dialog': Mode.dialog({
+    host,
+    renderer: Dialog.popup(),
+  }),
+  relay: Mode.relay(),
+} as { [K in ModeType]: Mode.Mode }
+
+export const porto = Porto.create({
+  ...PortoConfig.config[env],
+  mode: modes.relay,
+})
```

Comment:
Can't say I am a massive fan of this change. The `setMode` mechanism was kinda made for debugging on playground. 

What if `mode` could also be a function that accepted RPC methods as a param?

```suggestion
const config = Porto.porto._internal.config

export const porto = Porto.create({
  ...config,
  mode({ method }) {
    const dialogMethods = ['experimental_addFunds', 'wallet_sendCalls', ...]
    if (dialogMethods.includes(method)) return Mode.dialog(...)
    return config.mode
  },
})
```

Reply:
good idea. Will do in the next PR